recipes-graphics/vulkan/vulkan-wsi.bb

```diff
@@ -0,0 +1,52 @@
+DESCRIPTION = "Vulkan Window System Integration Layer"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=c2e771b72d60a13d2de384cb49055d00"
+DEPENDS = "libdrm vulkan-loader"
+
+SRC_URI = "git://gitlab.freedesktop.org/mesa/vulkan-wsi-layer.git;protocol=https;branch=main \
+           file://0001-MGS-6801-ccc-vkmark-on-wayland.patch \
+           file://0002-MGS-6823-nxp-Add-support-of-VK_COMPOSITE_ALPHA_OPAQU.patch \
+           file://0003-Update-minimum-version-of-CMake.patch"
+SRCREV = "cb1a50cf7e640ad7306e673131ded98c0f133628"
```

Comment:
You can use the vendor's suffix, as this is not a vendor-specific recipe. We must allow for its use in later layers and even in the core.